### DISCUSSION TITLE
TypeDORM State/Maintenance

### DISCUSSION THREAD

**Discussion by alalama** [typedorm/typedorm#381] [Q&A]
Hello TypeDORM community,

I am interested in using TypeDORM and would like to know if this repository is still being actively maintained. I noticed that there haven't been any recent updates and wanted to ask before potentially using it in my project or considering contributing.

Any information regarding the maintenance status or future plans would be greatly appreciated.

Thank you for your work on this project!

**Comment by Josecc** [comment]
Looking at the issues and commits there doesn't seem to be much traction. Even [contributions](https://github.com/typedorm/typedorm/pull/373) haven't been looked at in months :(
